src/ontology/cl-edit.owl

```diff
@@ -29154,6 +29155,18 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023120 "cochlear hair cell"
 AnnotationAssertion(rdfs:label obo:CL_4023120 "cochlea auditory hair cell")
 EquivalentClasses(obo:CL_4023120 ObjectIntersectionOf(obo:CL_0000202 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002227)))
 
+# Class: obo:CL_4023121 (sst chodl GABAergic cortical interneuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:15845086"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:31209381") Annotation(oboInOwl:hasDbXref "PMID:33372656") obo:IAO_0000115 obo:CL_4023121 "A sst GABAergic cortical interneuron that also expresses Chodl. These neurons are rare and correspond to the only known cortical interneurons with long-range projection.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023121 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasBroadSynonym obo:CL_4023121 "long-range GABAergic interneuron")
+AnnotationAssertion(rdfs:label obo:CL_4023121 "sst chodl GABAergic cortical interneuron")
+SubClassOf(obo:CL_4023121 obo:CL_0000099)
+SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956))
+SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
+SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000005444))
+SubClassOf(obo:CL_4023121 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015665))
```

Comment:
Could potentially make this an equivalent Class def. I think you have N&S clauses.

Reply:
I did think about this, however, I didn't feel comfortable doing that - might be me being paranoid but cause SST Chodl does not have SST in NS-forest, but instead Nos1 and Chodl, I wasn't super confident in making that and equiv class